src/components/views/userProfile/donationsTab/recurringTab/StreamRow.tsx

```diff
@@ -57,12 +64,12 @@ export const StreamRow: FC<IStreamRowProps> = ({ tokenStream }) => {
 							)
 						: '0'}
 				</P>
-				<P>{symbol}</P>
+				<P>{superToken?.symbol}</P>
 			</TableCell>
 			<TableCell>
 				<P color={semanticColors.jade[500]}>
-					{limitFraction(
-						formatUnits(monthlyFlowRate || 0n, decimals),
+					{formatDonation(
+						limitFraction(formatUnits(BigInt(monthlyFlowRate), 18)),
```

Comment:
@kkatusic found like 3 more instances where you format the units directly with 18, shouldn't this be related to the supertoken?

Reply:
that was the problem in first way, we can't use donation amount or better say flow rate amount for token decimals on 6 base, because flow will fail. We need to use super token decimal base :(